fastseq/optimizer/fairseq/beam_search_optimizer.py

```diff
@@ -47,7 +59,7 @@ def step(self, step, lprobs, scores):
         self.indices_buf.fmod_(vocab_size)
         return self.scores_buf, self.indices_buf, self.beams_buf
 
-@replace(TransformerEncoder)
+@replace(TransformerEncoder, not use_el_attn)
```

Comment:
How about using a global variable `USE_OPTIMIZED_CACHE_ATTN = not config.USE_EL_ATTN` instead of `not use_el_attn` to improve the readability.

Reply:
done